lib/util/load_rules.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/services.dart' show rootBundle;
+
+String get _rulesPath {
+  final path = "assets/text/rules.txt";
+  return path;
+}
```

Comment:
Could inline as `String get _rulesPath => "assets/text/rules.txt";`

Reply:
Yep, still not used to that! Hopefully won't have to expand it all out again in future.